src/test/java/com/ieltswise/controller/CommentControllerTest.java

```diff
@@ -0,0 +1,143 @@
+package com.ieltswise.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ieltswise.controller.request.StudentCommentRequest;
+import com.ieltswise.entity.StudentComment;
+import com.ieltswise.exception.EmailNotFoundException;
+import com.ieltswise.exception.NoPurchasedLessonsException;
+import com.ieltswise.service.CommentService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(CommentController.class)
+public class CommentControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private CommentService commentService;
+
+    StudentComment studentComment;
+
+    @BeforeEach
+    void setUp() {
+        studentComment = StudentComment.builder()
+                .id(1L)
+                .name("Bob")
+                .email("Bob001@gmail.com")
+                .value("I like learning English.")
+                .created(LocalDateTime.of(2024, 5, 2, 20, 19, 35, 0))
+                .build();
+    }
+
+    @Test
+    public void CommentController_GetAllComments_ReturnAllComments() throws Exception {
```

Comment:
You need to follow the method naming style. In our case, this is a camelcase and everything unnecessary needs to be removed:
testGetAllCommentsReturnAllComments

Reply:
fixed it